src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,12 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+
+import React from "react";
+
+import Todo from "./Todo";
+
+const TodoList = ({ todos }) => {
+  return todos.map(({ task, id }) => <Todo key={id} task={task} />);
+};
```

Comment:
I love the destructing it makes it more semantic for yourself coming back and working on the code. Great job getting the reps in, you will soon be the destructing expert!